### PR TITLE
Add wrapper for Gripper Camera Param python client

### PR DESCRIPTION
**Pull request by abaker-bdai**
This PR adds a wrapper for the Gripper Camera Param python client to `cam_wrapper.py`. This is a dependency for [this PR to spot_ros2](https://github.com/bdaiinstitute/spot_ros2/pull/138) that adds ros2 services for getting and setting gripper camera params.

### CODE REVIEW

**Review by heuristicus — COMMENTED** [review]
This should be in the image module, not the cam wrapper which is used for the spot cam. Unless I'm misunderstanding the contents.

**Comment by abaker-bdai** [comment]
> This should be in the image module spot_images, not the cam wrapper which is used for the spot cam. Unless I'm misunderstanding the contents.

This simply wraps the Gripper Camera Param Python client so the client can be exposed in the spot_ros2 interface via services. I assumed it went here instead of spot_images because it isn't actually capturing image data, just configuring the gripper camera settings.

**Comment by heuristicus** [comment]
In that case it might be best for it to go in the arm module. It definitely doesn't fit in the cam_wrapper.

File: spot_wrapper/wrapper.py [hunk 1]
```diff
@@ -616,12 +608,23 @@ def __init__(
                 MAX_COMMAND_DURATION,
                 self._claim_decorator,
             )
+
+            self._spot_images = SpotImages(
+                self._robot,
+                self._logger,
+                self._image_client,
+                self._rgb_cameras,
```
[the remark this answers is not in the document]
Comment (abaker-bdai) [reply spot_wrapper/wrapper.py]: Doh, you're right. Should be fixed now

**Pull request merged**